pkgs/development/compilers/dotnet/combine-packages.nix

```diff
@@ -1,20 +1,34 @@
 packages:
-{ buildEnv, lib }:
+{ buildEnv, makeWrapper, lib }:
+# TODO: Rethink how we determine and/or get the CLI.
+#       Possible options raised in #187118:
+#         1. A separate argument for the CLI (as suggested by IvarWithoutBones
+#         2. Use the highest version SDK for the CLI (as suggested by GGG)
+#         3. Something else?
 let cli = builtins.head packages;
 in
-assert lib.assertMsg ((builtins.length packages) != 0)
+assert lib.assertMsg ((builtins.length packages) < 1)
```

Comment:
This should be `> 1`, if you do `< 1` then this is only valid if you don't pass any packages!

Reply:
Oh right, my bad, will make a PR immediately